frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form-status.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import cx from 'classnames';
+import {
+  Title,
+  Button,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+  Bullseye,
+  Progress,
+  Stack,
+  StackItem,
+  Alert,
+  AlertVariant,
+  Spinner,
+} from '@patternfly/react-core';
+import { ErrorCircleOIcon, InProgressIcon } from '@patternfly/react-icons';
+import { DataUpload } from '../cdi-upload-provider';
+import { getProgressVariant } from '../upload-pvc-popover';
+import { killUploadPVC } from '../../../k8s/requests/cdi-upload/cdi-upload-requests';
+import { UPLOAD_STATUS } from '../consts';
+
+export const UploadPVCFormStatus: React.FC<UploadPVCFormStatusProps> = ({
+  upload,
+  isSubmitting,
+  isAllocating,
+  allocateError,
+  onErrorClick,
+  onSuccessClick,
+  onCancelFinish,
+}) => {
+  const [error, setError] = React.useState(allocateError || upload?.uploadError?.message);
+
+  React.useEffect(() => {
+    const newError = allocateError || upload?.uploadError?.message;
+    setError(newError);
+  }, [allocateError, upload]);
+
+  const getStatus = () => {
+    if (error) return <ErrorStatus error={error} onErrorPrimaryClick={onErrorClick} />;
+    if (isAllocating) return <AllocatingStatus />;
+    if (upload?.uploadStatus === UPLOAD_STATUS.CANCELED) return <CancellingStatus />;
+    return (
+      <UploadingStatus
+        upload={upload}
+        onSuccessClick={onSuccessClick}
+        onCancelFinish={onCancelFinish}
+        setError={setError}
+      />
+    );
+  };
+
+  return (
+    <Bullseye
+      className={cx({
+        'kv--create-upload__hide': !isSubmitting,
```

Comment:
just dont render if submitting ? Why using a css for that

Reply:
because I want to save the state of the form incase theres an error in the submitting view, and the user needs to apply a quick fix :)

Reply:
which state ? I see only `error` state here, you mean that one ?